src/openms/source/ANALYSIS/QUANTITATION/PeptideAndProteinQuant.cpp

```diff
@@ -370,21 +388,31 @@ namespace OpenMS
 
     // for (auto & a : accession_to_leader) { std::cout << a.first << "\tis led by:\t" << a.second << endl; }
 
+    set<String> unique_peptide_sequences;
     for (auto const& pep_q : pep_quant_)
     {
       String accession = getAccession_(pep_q.second.accessions,
                                        accession_to_leader);
       OPENMS_LOG_DEBUG << "Peptide id mapped to leader: " << accession << endl;
-      if (!accession.empty()) // proteotypic peptide
+
+      // not enough evidence or mapping to multiple groups
+      if (accession.empty()) continue;
+
+      // proteotypic peptide
```

Comment:
here it says proteotypic peptide but I guess it is proteogrouptypic ;)

Reply:
I think proteogrouptypic is the better measure. For singletons proteotypic = proteogrouptypic and for groups proteotypic would be 0 anyway, so the way you are counting here just adds information but does not lose any.